src/components/policy/policy_regular/include/policy/cache_manager.h

```diff
@@ -571,6 +571,12 @@ class CacheManager : public CacheManagerInterface {
                    const std::string& wers_country_code,
                    const std::string& language);
 
+  /**
+   * @brief Get information about last ccpu_version from PT
+   * @return ccpu_version from PT
+   */
+  const std::string GetCCPUVersionFromPT() const;
```

Comment:
@ydementieiev What is the sense to return by const **value**?

Reply:
@AByzhynar  done